app/views/groups/index.html.erb

```diff
@@ -1,14 +1,19 @@
 <p style="color: green"><%= notice %></p>
 
-<h1>Groups</h1>
+<h1>Categories</h1>
 
 <div id="groups">
+ <table class= 'group-table'>
+    <tr>
+      <th>Icon</th>
+      <th>Name</th>
+      <th>Total Spent</th>
+    </tr>
   <% @groups.each do |group| %>
     <%= render group %>
-    <p>
-      <%= link_to "Show this group", group %>
-    </p>
-  <% end %>
+  <%end%>
+ </table>
+</div>
+<div class= 'new-category'>
+<%= link_to "New category", new_group_path %>
 </div>
-
-<%= link_to "New group", new_group_path %>
```

Comment:
- [ ] it would be better if you stick to the design in order to make you app better 
- the background should be gray 
- the cards should be white
- the add new category should be take the whole width 
- the Design 
<img width="274" alt="Screen Shot 2022-10-14 at 19 44 02" src="https://user-images.githubusercontent.com/61790587/195919711-cd3cd69a-b39d-4350-a8a4-8a10fae66194.png">
- Yours 
<img width="390" alt="Screen Shot 2022-10-14 at 19 48 26" src="https://user-images.githubusercontent.com/61790587/195919731-278a7ed9-4ce0-4e4e-a5d2-5f1122981db5.png">